github/resource_github_repository_ruleset_test.go

```diff
@@ -363,6 +363,82 @@ func TestGithubRepositoryRulesets(t *testing.T) {
 		})
 
 	})
+	t.Run("Creates repository rulesets with paid features without errors", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("Creates a push repository ruleset without errors", func(t *testing.T) {
```

There are other paid features not covered by this test.